buildstockbatch/gcp/gcp.py

```diff
@@ -249,8 +428,11 @@ def main():
         },
     })
     print(GcpBatch.LOGO)
-    if 'GCP_BATCH_JOB_ARRAY_INDEX' in os.environ:
-        job_id = int(os.environ['GCP_BATCH_JOB_ARRAY_INDEX'])
+    if 'BATCH_TASK_INDEX' in os.environ:
+        # If this var exists, we're inside a single batch task.
+        # TODO: Would it be cleaner to move this to a main() in another file?
```

Comment:
I propose omitting potential improvements TODOS. There are a _ton_ of possible improvements and getting to them is fairly speculative at this point (then there's the dimension of sharing such improvements with the AWS, Eagle, and local scripts). Meanwhile, they're diluting the TODOs we really need to do.